src/actions/index.js

```diff
@@ -0,0 +1,16 @@
+import { REMOVE_BOOK, CREATE_BOOK } from '../constant/ActionTypes';
+
+export const createBook = book => ({
+  type: CREATE_BOOK,
+  book,
+});
+
+export const removeBook = id => ({
+  type: REMOVE_BOOK,
+  id,
+});
+
+// export const filterBook = category => ({
+//   type: CHANGE_FILTER,
+//   category,
+// });
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))